dashboard/client/pages/bazelci.tsx

```diff
@@ -0,0 +1,341 @@
+          >
+            <option value={-1}>Past month</option>
+            <option value={-3}>Past 3 months</option>
+            <option value={-6}>Past 6 months</option>
+            <option value={-9}>Past 9 months</option>
+            <option value={-12}>Past 12 months</option>
+            <option value={-24}>Past 24 months</option>
+            <option value={0}>All</option>
+          </select>
+        </div>
+      </div>
+      <Chart
+        org={param.org}
+        pipeline={param.pipeline}
+        data={data}
+        domain={domain}
+        excludeWaitTime={excludeWaitTime}
+      />
+    </div>
+  );
+}
+
+function JobStats({
+  param,
+  domain,
+  excludeWaitTime,
+}: {
+  param: StatsParam;
+  domain: number[] | undefined;
+  excludeWaitTime: boolean;
+}) {
+  const stats = useBuildkiteJobStats(param.org, param.pipeline, {
+    branch: param.branch,
+    from: param.from,
+  });
+
+  const sortedGroup = useMemo(() => {
+    if (stats.loading || stats.error) {
+      return [];
+    }
+
+    const data = stats.data.items;
+    var group = _.groupBy(data, (item) => item.bazelCITask);
+
+    return _.map(
+      _.sortBy(
+        _.map(group, (data) => {
+          const max = _.max(
+            _.map(data, (item) =>
+              excludeWaitTime ? item.runTime : item.waitTime + item.runTime
+            )
+          );
+          return { sortKey: -(max || 0), data: data };
+        }),
+        (item) => item.sortKey
+      ),
+      (item) => item.data
+    );
+  }, [stats, excludeWaitTime]);
+
+  if (stats.loading || stats.error || domain === undefined) {
+    return <></>;
+  }
+
+  return (
+    <>
+      {_.map(sortedGroup, (data) => (
+        <div
+          key={data[0].bazelCITask}
+          className="flex flex-col border shadow rounded bg-white ring-1 ring-black ring-opacity-5 flex-auto"
+        >
+          <div className="bg-gray-100 flex flex-row items-center border-b">
+            <span className="px-4 py-2 text-base font-medium">
+              {data[0].bazelCITask} | {data[0].name}
+            </span>
+          </div>
+          <Chart
+            org={param.org}
+            pipeline={param.pipeline}
+            data={data}
+            domain={domain}
+            excludeWaitTime={excludeWaitTime}
+          />
+        </div>
+      ))}
+    </>
+  );
+}
+
+export default function Page() {
+  const [domain, setDomain] = useState<number[]>();
+  const [monthOffset, setMonthOffset] = useState<number>(-1);
+  const [excludeWaitTime, setExcludeWaitTime] = useState<boolean>(true);
+
+  const param = useMemo(() => {
+    return {
+      org: "bazel",
+      pipeline: "bazel-bazel",
+      branch: "master",
+      from:
+        monthOffset < 0
+          ? DateTime.now().minus({ month: -monthOffset }).toISO()
+          : DateTime.fromSeconds(0).toISO(),
+    };
+  }, [monthOffset]);
+
+  return (
+    <Layout>
+      <div className="m-8 flex flex-col gap-8">
+        <BuildStats
```

Comment:
Perhaps add some explanation here that this reflects the times of successful builds in postsubmit (with a link)?